common/environment/setup/install.sh

```diff
@@ -265,3 +265,28 @@ _vcompletion() {
 			;;
 	esac
 }
+
+_vterminfo() {
```

Comment:
hm I think i like the idea of this but it should be in its own pr, which adds the function and changes all the other packages that compile terminfos manually as well as adding docs in manual.md

Reply:
Ok!